### DISCUSSION TITLE
Multi day reminders

### DISCUSSION THREAD

**Discussion by Stuntflyer** [mikeberger/borg_calendar#165] [Q&A]
I just installed the latest version of BORG calendar. Is there a way to set a "to do" reminder 1 or 2 days before the event?

Mike

**Comment by mikeberger** [comment]
You may have seen that for timed events, you can select when to popup a reminder per event:

![image](https://user-images.githubusercontent.com/6563665/183260929-e417307c-ed9c-4c33-a726-9b23a0b5db62.png)

Untimed todos also use this list of times, but in a different way. An untimed todo will start popping up at the largest reminder time interval before midnight of the todo date, and then keep popping every N minutes, which is a tunable option that defaults to 30 minutes.

You can also change the list of selectable popup times in the options:

![image](https://user-images.githubusercontent.com/6563665/183261069-d8393fd8-5288-4371-a762-a75cf1546223.png)

I personally set the largest popup time to be 1440 minutes (24 hours) before the event time. The default is 6 hours. If I have this time selected for an untimed todo, it will first pop up 24 hours before midnight of the todo date. You can set any times here. You can pop up reminders weeks before the event if you want to.

**Reply by Stuntflyer** [reply]
Thanks,

If I set a 1440 and 2880 reminder time and check both, will I get a reminder 2 days before and 1 day before?

**Comment by mikeberger** [comment]
For timed events, yes.

For untimed todos, reminders would start 2 days before and keep popping every 30 minutes, unless hidden.

**Reply by Stuntflyer** [reply]
Perfect, thanks!
